src/api/api.ts

```diff
@@ -17,6 +18,48 @@ export function saveEncounter(abortController: AbortController, payload, encount
   });
 }
 
+export function saveAttachment(patientUuid, content, conceptUuid, date, encounterUUID, abortController) {
+  const url = '/ws/rest/v1/attachment';
+
+  const formData = new FormData();
+
+  const cameraUploadType = content
+    ?.split(';')[0]
+    .split(':')[1]
+    .split('/')[1];
+
+  const fileCaption = typeof content === 'object' ? content.name : `camera-upload.${cameraUploadType}`;
+
+  formData.append('fileCaption', fileCaption);
+  formData.append('patient', patientUuid);
+
+  if (typeof content === 'object') {
+    formData.append('file', content);
+  } else {
+    formData.append('file', new File([''], 'OHRIFileConceptTest.png'), 'OHRIFileConceptTest.png');
+    formData.append('base64Content', content);
+  }
+  formData.append(
+    'json',
+    JSON.stringify({
+      person: patientUuid,
+      concept: conceptUuid,
+      groupMembers: [],
+      obsDatetime: date,
+      encounter: encounterUUID,
+    }),
+  );
+
+  return openmrsFetch(url, {
+    method: 'POST',
+    // headers: {
+    //   'Content-Type': 'multipart/form-data',
```

Comment:
Is there a specific reason why your request has no headers?

Reply:
Adding headers was corrupting the request, I think it automatically assigns them.